validator/validators/performance/performance_validator.go

```diff
@@ -48,7 +48,7 @@ func (s *PerformanceValidator) GenerateLoad() (err error) {
 		agentConfigFilePath   = s.vConfig.GetCloudWatchAgentConfigPath()
 		dataType              = s.vConfig.GetDataType()
 		dataRate              = s.vConfig.GetDataRate()
-		receiver              = s.vConfig.GetPluginsConfig()
+		receiver              = s.vConfig.GetPluginsConfig()[0] //Assuming one plugin at a time
```

Comment:
If this only uses index 0, then when will the other plugins get validated? Why bother passing in plugins that the function does not need?